SupportClient/SupportClient/Controllers/ViewController.swift

```diff
@@ -0,0 +1,198 @@
+//
+//  ViewController.swift
+//  SupportClient
+//
+//  Created by Omar Rasheed on 2/15/20.
+//  Copyright © 2020 Cornell Appdev. All rights reserved.
+//
+
+import UIKit
+import SnapKit
+
+class ViewController: UIViewController {
+    
+    
+    var newConversationButton: UIButton!
+    var feedbackLabel: UILabel!
+    var searchController: UISearchController! // TODO: For later
+    var feedbackTableView: UITableView!
+        
+    var feedbackData: [Feedback]!
+    var filteredFeedbackData = [Feedback]() // TODO: For later
+    
+    var countEditTaps: Int = 0
+    var isFeedback: Bool!
+    let cellHeight: CGFloat = 75
+    let reuseID: String = "FeedbackTableViewCell"
+
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        view.backgroundColor = Constants.backgroundColor
+        // Do any additional setup after loading the view.
+        
+        isFeedback = setUpData() == 0 ? false : true
+        setUpNavigationBar()
+        setUpFeedbackLabel()
+        setUpConversationButton()
+        setUpFeedbackTableView()
+        setUpConstraints()
+        
+    }
+    
+    // TODO: hook up to actual server to load the data
+    // This is dummy data for testing
+    // Add variables to feedbackData to show the tableView + editing functionality
+    func setUpData() -> Int {
+        
+        let _ = Feedback(title: "Ithaca Transit Bug", message: "This app sometimes glitches out on me and shows the wrong bus times", hasUnread: true)
+        let _ = Feedback(title: "chatOS Idea", message: "what if admin responed via macOS applications and not iOS applications??", hasUnread: false)
+        feedbackData = []
+        return feedbackData.count
+        
+    }
+    
+    func setUpNavigationBar() {
+        
+        navigationController?.navigationBar.barTintColor = Constants.navigationTintColor
+        
+        let attributedTitle = NSAttributedString(string: "Feedback", attributes: [NSAttributedString.Key.font: UIFont(name: Constants.robotoMedium, size: 21)!, NSAttributedString.Key.foregroundColor: Constants.titleColor])
```

Comment:
we should also declare all of our fonts in a `UIFont+Extensions.swift` file, similar to this: https://github.com/cuappdev/uplift-ios/blob/master/Uplift/Extensions/UIFont%2BShared.swift